addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1224,6 +1247,21 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
         const dependencyName = fieldEl.dataset.visibilityDependency;
         return this.formEl.querySelector(`.s_website_form_input[name="${dependencyName}"]`);
     },
+    /**
+     * Returns the other field labels of the form.
+     *
+     * @returns {String[]} The other field labels of the form.
+     */
+    _getListOfOtherLabels() {
+        const formEl = this.$target[0].closest(".s_website_form");
+        const otherFieldLabels = [];
+        for (const fieldEl of formEl.querySelectorAll(".s_website_form_field")) {
+            if (fieldEl !== this.$target[0]) {
+                otherFieldLabels.push(fieldEl.querySelector(".s_website_form_input").name);
+            }
+        }
+        return otherFieldLabels;
+    },
```

Comment:
```suggestion
_getListOfOtherLabels() {
    const fieldEls = Array.from(document.querySelectorAll(".s_website_form_field"));
    const otherFieldLabels = fieldEls
        .filter(fieldEl => fieldEl !== this.$target[0])
        .map(fieldEl => fieldEl.querySelector(".s_website_form_input").name);
    return otherFieldLabels;
}
```

Reply:
@bvr-odoo, I also thought to use the diff you are proposing but the drawback of it it that you have to browse twice the `fieldEls` array (once for the  `filter()` and then for the `map()`) compare to only one with the current solution. Something I could do is to directly loop on `document.querySelectorAll(".s_website_form_field")` however, the drawback of it is that we have to perform a `querySelector` on the entire document so that is the reason why I chose to first find the `formEl` :relaxed: What do you think?

Reply:
You are right about the fact that we have to avoid to perform a `querySelector` on the entire document. I just forget to use the line  `const formEl = this.$target[0].closest(".s_website_form");` in my suggestion ;) .
My goal here using map and filter was to make the code cleaner and I think for the performance it doesn't change anything in this case.

But it was only a suggestion so do what you want ;)